ompi/mca/coll/base/coll_base_util.c

```diff
@@ -558,3 +559,69 @@ const char* mca_coll_base_colltype_to_str(int collid)
     }
     return colltype_translation_table[collid];
 }
+
+OBJ_CLASS_INSTANCE(ompi_coll_base_hostname_item_t, opal_list_item_t, NULL, NULL);
```

Comment:
Do you really need a class here?  I think you could probably use a `char**` here, and the `opal_argv` functions.  It may not be a huge difference, but the `opal_argv` functions might be slightly lighter-weight...?